src/lib.rs

```diff
@@ -111,3 +111,13 @@ pub use frame_buffer::{FrameBuffer, FrameBufferMut};
 pub use header::{Header, Envmap};
 pub use input::InputFile;
 pub use output::ScanlineOutputFile;
+
+/// Set the number of worker threads to use for doing I/O.
+///
+/// If set to 0, multi-threaded I/O is disabled.
+pub fn set_global_thread_count(thread_count: i32) {
```

Comment:
The C++ API takes an `int`, so I've used an `i32` here, but it would probably make more sense to be an `u32` I guess... Not sure if I should change to `u32` and cast it?

Reply:
Yeah, I think it probably makes sense to do `u32`, and then check to make sure it fits in the `c_int`.  Remember that C ints aren't guaranteed to be 32 bit, so you'll want to do some kind of checked conversion.  I assume there's something in `std` for that, but not actually sure.

Reply:
Oh, and I think straight in `lib.rs` is fine.

Nit:

According the documentation in the [C++ header](https://github.com/openexr/openexr/blob/develop/OpenEXR/IlmImf/ImfThreading.h), the threads aren't used for I/O, they're used for compression/decompression.  Only one thread does the actual I/O.  So I'd change the doc-comment to reflect that.

Reply:
There is `TryFrom` for doing checked conversion but it's unstable... Looking at the source, it just checks the value against the max value of the target type, so I'll do the same thing with an `assert!()`.